SQL/0000-00-05-ElectrophysiologyTables.sql

```diff
@@ -201,13 +200,94 @@ CREATE TABLE `physiological_archive` (
+    PRIMARY KEY (`FileType`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create physiological_annotation_file table
+CREATE TABLE `physiological_annotation_file` (
+    `AnnotationFileID`    INT(10)      UNSIGNED NOT NULL AUTO_INCREMENT,
+    `PhysiologicalFileID` INT(10)      UNSIGNED NOT NULL,
+    `FileType`            VARCHAR(20)  NOT NULL,
+    `FilePath`            VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`AnnotationFileID`),
+    CONSTRAINT `FK_phys_file_ID`
+        FOREIGN KEY (`PhysiologicalFileID`)
+        REFERENCES `physiological_file` (`PhysiologicalFileID`),
+    CONSTRAINT `FK_annotation_file_type`
+        FOREIGN KEY (`FileType`)
+        REFERENCES `physiological_annotation_file_type` (`FileType`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create annotation_archive which will store archives of all the annotation files for
+-- Front-end download
+CREATE TABLE `physiological_annotation_archive` (
+  `AnnotationArchiveID` INT(10) UNSIGNED NOT NULL   AUTO_INCREMENT,
+  `PhysiologicalFileID` INT(10) UNSIGNED NOT NULL,
+  `Blake2bHash`         VARCHAR(128)     NOT NULL,
+  `FilePath`            VARCHAR(255)     NOT NULL,
+  PRIMARY KEY (`AnnotationArchiveID`),
+  CONSTRAINT `FK_physiological_file_ID`
+    FOREIGN KEY (`PhysiologicalFileID`)
+    REFERENCES `physiological_file` (`PhysiologicalFileID`)
+    ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create annotation_parameter table
+-- Note: This corresponds with the JSON annotation files
+CREATE TABLE `physiological_annotation_parameter` (
+    `AnnotationParameterID` INT(10)      UNSIGNED NOT NULL AUTO_INCREMENT,
+    `AnnotationFileID`      INT(10)      UNSIGNED NOT NULL,
+    `Sources`               VARCHAR(255),
+    `Author`                VARCHAR(50),
+    PRIMARY KEY (`AnnotationParameterID`),
+    CONSTRAINT `FK_annotation_file_ID`
+        FOREIGN KEY (`AnnotationFileID`)
+        REFERENCES `physiological_annotation_file` (`AnnotationFileID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create an annotation_label_type table
+CREATE TABLE `physiological_annotation_label` (
+  `AnnotationLabelID`    INT(5)       UNSIGNED NOT NULL      AUTO_INCREMENT,
+  `LabelName`            VARCHAR(255)          NOT NULL      UNIQUE,
+  `LabelDescription`         VARCHAR(255)          DEFAULT NULL,
+  PRIMARY KEY (`AnnotationLabelID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create annotation_tsv table
+-- Note: This corresponds with the .tsv annotation files
+CREATE TABLE `physiological_annotation_instance` (
+    `AnnotationInstanceID`  INT(10)        UNSIGNED NOT NULL AUTO_INCREMENT,
+    `AnnotationFileID`      INT(10)        UNSIGNED NOT NULL,
+    `AnnotationParameterID` INT(10)        UNSIGNED NOT NULL,
+    `Onset`                 DECIMAL(10, 3)          NOT NULL,
+    `Duration`              INT(10)                 NOT NULL DEFAULT 0,
+    `AnnotationLabelID` INT(5)         UNSIGNED NOT NULL,
+    `Channel`               VARCHAR(255),
```

Comment:
Channel should be Channels (it can store a collection of channels). And a VARCHAR was not enough to store the entries of my sample, I had to change it for a text.